README.md

```diff
@@ -1,6 +1,6 @@
-# OpenCensus service
+# OpenCensus Service
 
-OpenCensus service is an experimental component that collects and
+OpenCensus Service is an experimental component that collects and
 exports from the OpenCensus instrumented processes available from the
```

Comment:
Mention the support for other formats, as well as the fact that we support multiple signals (metrics).